packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -1480,281 +992,141 @@ describe('NetworkController', () => {
   });
 
   describe('setProviderType', () => {
-    [NetworkType.mainnet, NetworkType.goerli, NetworkType.sepolia].forEach(
-      (networkType) => {
-        describe(`given a network type of "${networkType}"`, () => {
-          it('updates the provider config in state with the network type, the corresponding chain ID, and a special ticker, clearing any existing RPC target and nickname', async () => {
-            const messenger = buildMessenger();
-            await withController(
-              {
-                messenger,
-                state: {
-                  providerConfig: {
-                    type: NetworkType.localhost,
-                    rpcTarget: 'http://somethingexisting.com',
-                    chainId: '99999',
-                    ticker: 'something existing',
-                    nickname: 'something existing',
-                  },
+    for (const { networkType } of INFURA_NETWORKS) {
```

Comment:
This change is strictly not necessary, but I had to use `INFURA_NETWORKS` for another test, so I figured I would expand the shape of that array to match the extension, and then update its usage here to also match the extension.